tslint.json

```diff
@@ -1,3 +1,11 @@
 {
-  "extends": "gts/tslint.json"
+  "extends": "gts/tslint.json",
+  "rules": {
```

Comment:
AFAIK we don't use these rules anywhere else. Any reason why we don't just follow suit with how the other repos are configured?

Reply:
Oh, because lint will throw a ton of errors otherwise, right? I'd like to get input from @JustinBeckwith but if we add all these options we should create a tracking issue to remove them and probably create a PR per option that addresses the errors thrown by said option.

Reply:
Yep, more than a ton. I was planning on tackling some of them as I go through this noImplicitAny process, leaving the issues that require I rework logic to the better judgment of those with the poder (you guys). And the tracking issue sounds like a good idea

Reply:
@callmehiphop should I go ahead and make the tracking issue myself (if my account has permission)?

Reply:
I'm ok with this, as long as we have an issue created to come back around and fix it.

Reply:
@joecervino yeah go for it! If you get blocked my perms let me know and I'll label it up.

Reply:
@callmehiphop Sweet, filed it, just needs labeling